baseapp/baseapp.go

```diff
@@ -216,6 +216,9 @@ func (app *BaseApp) MountStores(keys ...storetypes.StoreKey) {
 		case *storetypes.TransientStoreKey:
 			app.MountStore(key, storetypes.StoreTypeTransient)
 
+		case *storetypes.MemoryStoreKey:
```

Comment:
Not sure if adding this to baseapp is the way to go

Reply:
What's the context here?

Reply:
runtime/builder uses baseApps' `MountStores`. Without this addition we get this error `Unrecognized store key type :*types.MemoryStoreKey`.
I think that if we filter out any `MemoryStoreKey`s before calling this everything works just fine... I don't know enough about these things to know exactly what's up 😅 

https://github.com/cosmos/cosmos-sdk/blob/8eaff8fadd408762fa5a3c5a40969bda4548057e/runtime/builder.go#L27-L41

Reply:
nvm tried again removing it and it doesn't work

Reply:
I'm not really sure about this code. I'd have to delegate to @aaronc here.